include/PianoRoll.h

```diff
@@ -294,7 +303,8 @@ protected slots:
 	int noteEditRight() const;
 	int noteEditLeft() const;
 
-	void dragNotes( int x, int y, bool alt, bool shift, bool ctrl );
+	void dragNotes( int x, int y, bool alt, bool shift, bool ctrl, Note * nt );
+	Note * m_draggedNote;
```

Comment:
```suggestion
	void dragNotes(int x, int y, bool alt, bool shift, bool ctrl);
```

From what I read in the code it seems that `m_draggedNote` is being used with the same goal as `m_currentNote`. Using the latter would probably fix the bug with dragging a note on top of another one (since `m_currentNote` is set on the `mousePressEvent` and will not change during `mouseMoveEvent`) and will prevent adding unnecessary variables to the method and class.